Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -73,10 +74,27 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
       selectedRewardFromId,
       refTag
     )
+    .filter { arg in
+      let (project, _, _) = arg
```

Comment:
you probably did this to help the compiler but i'm sure it'll also compile as `.filter { project, _, _ in ... }`